src/components/Carousel/index.ts

```diff
@@ -0,0 +1,3 @@
+import Carousel from './Carousel.vue';
```

Comment:
```suggestion
import Carousel from '@/components/Carousel.vue';
```

Reply:
Relative import is used for all other components too so leaving as is